src/qr.rs

```diff
@@ -30,6 +30,7 @@ const IDELTACHAT_NOSLASH_SCHEME: &str = "https://i.delta.chat#";
 const DCACCOUNT_SCHEME: &str = "DCACCOUNT:";
 pub(super) const DCLOGIN_SCHEME: &str = "DCLOGIN:";
 const DCWEBRTC_SCHEME: &str = "DCWEBRTC:";
+const TG_SOCKS_SCHEME: &str = "https://t.me/socks";
```

Comment:
I think it's better not to abbreviate TELEGRAM to TG, TG has lots of meanings. Though this is minor

Reply:
i am somehow hesitant to brand brands to the source code; "tg" is also not unusual in that area, even if it can mean different things, the context should be clear enough. as it is nothing on wire or api level, we can easily change that at any point